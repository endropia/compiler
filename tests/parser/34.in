var
    i: integer;
begin
    for i := 1  10 do
        writeln('*');
end.