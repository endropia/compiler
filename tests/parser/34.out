(4, 17) 'to' or 'downto' expected